### PR TITLE
Lma 120

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: VAS.Tests/Core/Serialization/TestObjectChangedParser.cs [hunk 1]
```diff
@@ -214,8 +214,8 @@ public void TestAllObjectsParsed ()
 		public void TestHotKeyModifierMigration ()
 		{
 			HotKey hotkey = new HotKey {
-				Key = (int)Keyboard.KeyvalFromName ("Up"),
-				Modifier = (int)Keyboard.KeyvalFromName ("Shift_L")
+				Key = (int)App.Current.Keyboard.KeyvalFromName ("Up"),
+				Modifier = (int)App.Current.Keyboard.KeyvalFromName ("Shift_L")
```
Comment (vguzmanp) [inline VAS.Tests/Core/Serialization/TestObjectChangedParser.cs]: `uint` to int?

File: VAS.Services/Controller/ProjectsController.cs [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+		void HandleImport (ImportEvent<TModel> evt)
+		{
+			// FIXME: should we be using this?
+		}
+
+		void HandleNew (CreateEvent<TModel> evt)
+		{
+			// FIXME: should we be using this?
+		}
```
Comment (vguzmanp) [inline VAS.Services/Controller/ProjectsController.cs]: I find it weird that the CreateEvent is not implemented...

File: VAS.Services/Controller/TemplatesController.cs [hunk 3]
```diff
@@ -103,33 +104,34 @@ public abstract class TemplatesController<T, W>: IController
...
-			App.Current.EventsBroker.Unsubscribe<CreateEvent<T>> (HandleNew);
-			App.Current.EventsBroker.Unsubscribe<DeleteEvent<T>> (HandleDelete);
-			App.Current.EventsBroker.Unsubscribe<ChangeNameEvent<T>> (HandleChangeName);
+			App.Current.EventsBroker.Unsubscribe<ExportEvent<TModel>> (HandleExport);
+			App.Current.EventsBroker.Unsubscribe<ImportEvent<TModel>> (HandleImport);
+			App.Current.EventsBroker.Unsubscribe<UpdateEvent<TModel>> (HandleSave);
+			App.Current.EventsBroker.Unsubscribe<CreateEvent<TModel>> (HandleNew);
+			App.Current.EventsBroker.Unsubscribe<DeleteEvent<TModel>> (HandleDelete);
+			App.Current.EventsBroker.Unsubscribe<ChangeNameEvent<TModel>> (HandleChangeName);
 			started = false;
```
Comment (vguzmanp) [inline VAS.Services/Controller/TemplatesController.cs]: Added an entry to the Coding Style about generic type naming. (https://fluendo.atlassian.net/wiki/display/Team/Coding+Style#CodingStyle-Generictypenaming)

File: VAS.Tests/Core/Common/TestRegistry.cs [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+	{
+	}
+
+	public class DummyObjectLow : IDummyObjectInterface
+	{
+	}
+
+	public class DummyObjectHigh : IDummyObjectInterface
+	{
+	}
```
Comment (aameijide) [inline VAS.Tests/Core/Common/TestRegistry.cs]: Move these classes to VAS.Tests.Utils

Comment (ylatuya) [inline VAS.Tests/Core/Common/TestRegistry.cs]: It's very specific to this test and only used there, that's why I didn't added it to Utils

**Comment by aameijide** [comment]
:+1:

**Pull request merged**